ecl/hqlcpp/hqlcerrors.hpp

```diff
@@ -386,7 +387,7 @@
 #define HQLERR_SetUnknownLength_Text            "Sets of items of unknown length are not yet supported!"
 #define HQLERR_HashStoredTypeMismatch_Text      "#STORED (%s) type mismatch (was '%s' replacement '%s')"
 #define HQLERR_CountAllSet_Text                 "Cannot count number of elements in ALL"
-#define HQLERR_OnlyKeyFixedField_Text           "Can only key fixed fields at fixed offsets"
+#define HQLERR_OnlyKeyFixedField_Text           "Cannot key field %s - only fixed fields at fixed offsets"
```

Comment:
Is this still true?

Reply:
It is for index fields.  I didn't want to dig into changing that for this PR, the aim of the change was to add the field name to the error message.